cnd/src/lib.rs

```diff
@@ -81,20 +81,25 @@ pub fn data_dir() -> Option<PathBuf> {
     ProjectDirs::from("", "", "comit").map(|proj_dirs| proj_dirs.data_dir().to_path_buf())
 }
 
+/// Returns `/Users/[username]/Library/Application Support/Lnd/`.
```

Comment:
Should we just inline this maybe?
If we use `cfg = macos` anyway, I don't really see a point in using generic code if we already know what we want it to be :)

Reply:
We use these functions in the tests now.

Reply:
Inline was probably the wrong word, I meant literally constructing the above path manually instead of delegating to `ProjectDirs`.

Reply:
I don't see the benefit, we already depend on the library that provides `ProjectDirs` so why not use it?  If you mean remove that dependency all together that's different, I can create a ticket if you want to do that.